pom.xml

```diff
@@ -49,7 +49,7 @@
       <tag>HEAD</tag>
   </scm>
     <properties>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <jenkins.version>1.609.2</jenkins.version>
```

Comment:
I've kept the override. I assume is the thing to do as the new 1.13 version keeps it.